### ISSUE TITLE
JFiles CommandLine Does Nothing After Initial Input

### ISSUE THREAD

**Issue by LoganRickert** [rsanchez-wsu/jfiles#227]
Branch: Master
Package: edu.wright.cs.jfiles.client
File: JFilesClient.java

Issue:

The commandline tool does nothing after you give it an initial input. It only writes the first line to output then flushes.

Example IO:

```
Establishing connection. Please wait ...
Connected: Socket[addr=localhost/127.0.0.1,port=9786,localport=56676]
> list
test
file
```

**Comment by LoganRickert** [comment]
Did some more further digging and testing and the server appears to receive the command okay. The issue seems to be that the client is not receiving the feedback from `JFilesServer.send(String)`. Unsure if the issue is that the `JFilesServer.send()` method is malfunctioning or the `JFilesClientThread.run()` method is malfunctioning.

**Comment by LoganRickert** [comment]
After digging further, it appears that the command is successfully sent to the server and processed. The  issue is that the JFilesClientThread.run() method is never called. The reason this happens is because the thread is never started. By adding `client.start();` below the declaration of the thread in `JFilesClient`, the problem is fixed.

``` JAVA
public void start() throws IOException {
	// ...
		client = new JFilesClientThread(this, socket);
		client.start(); // New line added
	// ...
}
```

The reason a new prompt is never shown is because the server sends the new prompt. This also fixes that issue.

**Comment by LoganRickert** [comment]
This issue has been resolved with commit b8f8445a91c3104cb3f2dfd596369a47ae1ab8ce
